### PR TITLE
Adapt trust relay sendmessage struct

### PR DESCRIPTION
**Pull request by dpereskokov**

### CODE REVIEW

File: js/src/Trust.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import {WalletLinkProvider} from "./WalletLinkProvider";
+
+export class Trust {
+    public static TrustWeb3Provider(
+        jsonRpcUrl: string,
+        chainId: number = 1,
+    ): WalletLinkProvider {
+        const trustRelay = new TrustRelay();
+        const walletLink = new WalletLink({
+            relay: trustRelay as unknown as WalletLinkRelay,
```
Comment (hewigovens) [inline js/src/Trust.ts]: looks weird, two `as`

**Comment by hewigovens** [comment]
let's setup CI too?

**Comment by dpereskokov** [comment]
@vikmeup could you please add project to azure so I can configure it?

**Comment by hewigovens** [comment]
@dpereskokov just try GitHub Action, it's easy

**Comment by vikmeup** [comment]
for public repos I would recommend to use either travis or github actions

**Comment by hewigovens** [comment]
CI added

**Pull request merged**